### ISSUE TITLE
Why we use "CocoCurated datasets"?

### ISSUE THREAD

**Issue by ghost** [xu-ji/IIC#75]
What happen if we don't use 'Coco - curated datasets'?

**Comment by xu-ji** [comment]
You can run the method on any image dataset. 
Coco curated is just a subset of Coco.
If you use pre-trained models on other datasets, you will be relying on generalisation from the original training dataset. So good performance is not guaranteed.

**Comment by ghost** [comment]
Oh thanks. I meant to ask Why don't we use the whole Coco datasets? Coco-curated is a subset of Coco as you said, and if we train on the whole datsets, then it might gain some more accuracy.

**Comment by xu-ji** [comment]
I think we did the subset to make sure for each image 75% of the pixels were relevant. Coco-stuff contains images from Coco so the proportion of "stuff" pixels varies, and we were focusing on textural stuff classes because of the local spatial invariance assumption.
